### ISSUE TITLE
Delay in Activity does not support more than 20 seconds. 

### ISSUE THREAD

**Issue by Seexelas** [unfoldedcircle/feature-and-bug-tracker#137]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Hi,

I have done a Zidoo activity where I need a delay of more than 20 secs to let my Zidoo media player start so I can after this set the input. 

It seems that a delay superior of 20 seconds freeze the activity start and the only way to exit is by power off or restart the remote. 

### How to Reproduce

Create an activity with different entities :
(in my case Panasonic Tv, then go to HDMI 2, then my AVR, input to Zidoo, then Zidoo starts and then a delay of 30 secs before setting the Zidoo to input on Home Theater.

Launch the activity

When the delay of 30 secs arrives, the activity freeze and do nothing. 

### Expected behavior

The activity starts with the predefine settings. 

### System version

1.4.2

### What part of the system affected by the problem?

Overall system

### Additional context

[UCR2_logs_2023-10-18.txt](https://github.com/unfoldedcircle/feature-and-bug-tracker/files/13033698/UCR2_logs_2023-10-18.txt)

**Comment by zehnm** [comment]
Thanks for reporting!
Indeed, there's a hard limit of 30 seconds in the activity execution (log entry: `started activity with timeout of 30s`) and the individual delay steps are not accounted for.
I've also discovered some UI issues while testing this.
We'll fix this.

**Comment by zehnm** [comment]
Release 1.4.3 increases the timeout from 30 to 60s as a quick "workaround". However, the UI might still show an error.
A real fix is in the works.

**Comment by thebatfink** [comment]
Where is this a couple of years later? Is the real fix still yet to be completed? I still hit the 60 seconds hard limit.

**Comment by rob5074** [comment]
its been almost 2 years and still no fix for this issue? If this issue isn't fixed that will make this remote useless unless its the most basic tasks!! This needs to be addressed ASAP

**Comment by thegame3202** [comment]
@zehnm - Any updates with this? Anyone with a projector typically uses a longer delay after turning ON the projector. My workaround for the time being is to turn on my projector with Home Assistant, wait 1 minute, then turn on the Remote 3 activity. But I'd prefer to have it all native on the remote for ease of use for other family members. Thanks!

**Comment by thebatfink** [comment]
I would also appreciate this resolution. Even if it could just be extended again to hard 90 or 120 seconds limit it would be very helpful.

**Comment by zehnm** [comment]
I'll have another look.

**Comment by thegame3202** [comment]
> I'll have another look.

Thank you!! I agree that even a 5 minute max would revolve most issues. But I’d be happy with 2 minutes if there was some limitation.

**Comment by thebatfink** [comment]
Thanks zehnm

**Comment by thegame3202** [comment]
Agreed! New 5 minute limit works great. I've moved all my automations over to the remote natively now :-) Thank you!!

**Comment by zehnm** [comment]
Closing: timeout is now 5 minutes since firmware 2.8.2
